src/applications/registry.json.md

```diff
@@ -2,6 +2,6 @@
 
 The [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/master/src/applications/registry.json) is used by the static site generator to create templates the match your applications; this file has been moved to the [content-build](https://github.com/department-of-veterans-affairs/content-build/) repository. Please make your updates in a feature branch (with the same name) over there.
 
-If you are trying to render a page to view or test a widget in this repository, you can add that information into the [registry.scaffold.json](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/registry.scaffold.json) file and it will generate locally and in CI with `yarn build:webpack --env.scaffold`
+If you are trying to render a page to view or test a widget in this repository, you can add that information into the [registry.scaffold.json](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/registry.scaffold.json) file and it will generate locally and in CI with `yarn build` or `yarn build:webpack --env.scaffold`
```

Comment:
nit
```suggestion
To render a page to view or test a widget in this repository, you can add that information into the [registry.scaffold.json](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/registry.scaffold.json) file and it will generate locally and in CI with `yarn build` or `yarn build:webpack --env.scaffold`
```